components/x-eventpromo/src/styles/bootstrap.scss

```diff
@@ -0,0 +1,10 @@
+@import "n-ui-foundations/main";
+@import 'o-buttons/main';
+
+$o-fonts-is-silent: false;
+@import 'o-fonts/main';
+
+$ep-standard-space: 20px;
+$ep-half-space: $ep-standard-space/2;
+
+@include oFontsInclude(MetricWeb, $weight: light);
```

Comment:
What is the intention of this bootstrap? If apps were to include this component should they include this? They will already have the fonts setup etc.

Reply:
Indeed, this may only be needed for storybook and demos so definitely something to review. 
So far I was literally trying to reproduce what we have in n-eventpromo like for like in x-dash, without too much consideration for the final integration.